source/documentation/archive/serving_data/__index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Fuseki: serving RDF data over HTTP"
+slug: index
+---
+
+## See the [Fuseki2 documentation](/documentation/fuseki2/).
+
+----
+> This page covers Fuseki1.
+> Fuseki1 is deprecated and has been retired.
+> [Fuseki2](/documentation/fuseki2/) is current.
+> <br/>
+> The last release of Jena with this module is Jena 3.9.0.
+> See [jena-fuseki1/README.md](https://github.com/apache/jena/tree/master/fuseki1).
+> The [original documentation](fuseki1).
```

Comment:
404 for me. Maybe `(fuseki1.html)`? (or maybe add some rewrite setting to handle when extension is omitted?)